wisdem/inputs/geometry_schema.yaml

```diff
@@ -3316,6 +3344,79 @@ definitions:
                 unit: N/m2
                 minItems: 2
                 uniqueItems: false
+        shearshear_stiff:
```

Comment:
these terms are not the terms we're proposing in windIO v2.0. is this mismatch intended?